code/modules/clothing/glasses/hud.dm

```diff
@@ -287,3 +295,4 @@
 	desc = "These sunglasses are special, and let you view potential criminals."
 	icon_state = "sun"
 	inhand_icon_state = "sunglasses"
+	flags_cover = GLASSESCOVERSEYES
```

Comment:
subtype of the spacecop glasses, which already has the flag
```suggestion
```

Reply:
knew at least a few where gona slip up